lib/teleterm/apiserver/apiserver.go

```diff
@@ -72,18 +83,21 @@ func newListener(hostAddr string) (net.Listener, error) {
 		return nil, trace.BadParameter("invalid host address: %s", hostAddr)
 	}
 
-	if uri.Scheme != "unix" {
-		return nil, trace.BadParameter("invalid unix socket address: %s", hostAddr)
-	}
-
-	lis, err := net.Listen(uri.Scheme, uri.Path)
+	lis, err := net.Listen(uri.Scheme, getAddrByScheme(uri))
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
 
 	return lis, nil
 }
 
+func getAddrByScheme(uri *url.URL) string {
```

Comment:
Can we just use TCP in all cases (and stop using Unix sockets completely)?

I think this might simplify things.

Reply:
I would like to use UDS everywhere, they don't require TLS, are simpler to configure and don't cause firewall prompts. If we can use UDS where it is possible, I'd do it.

BTW, I replaced `url.Parse` with `utils.ParseAddr` so `getAddrByScheme` is no longer needed.